Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
That's an interesting point around the Marlin G-code, speaking of standards and practices. `T` is correct for the tool number. However, there's some question whether that's cool with G-code standards. Specifically, as I understand it, the G-code guidelines we're following reserve `G`, `M`, and `T` for the main codes, `N` for line numbers, `F` for feed-rate, `X`, `Y`, and `Z` for the three primary axes, etc.

That said, these "standards" are pretty loosely applied in Marlin. The `T` code is used by `M200` for the extruder number. `M204` and `M205` use it for "travel moves" rate. `M206` uses it for "theta." And a few other codes, such as `M221`, use it for the extruder number.

But `M301` and `M303` use `E` for the extruder number. `G29` uses `E` as a flag for "enhanced G29."

So, for consistency, `T` seems best for the extruder number right now.